modules/nw-metallb-example-addresspool.adoc

```diff
@@ -62,3 +62,44 @@ spec:
   - 10.0.100.0/28
   - 2002:2:2::1-2002:2:2::100
 ----
+
+== Example: Assign IP address pools to services or namespaces
+You can assign IP addresses from an `IPAddressPool` to services and namespaces that you specify.
+
+If you assign a service or namespace to more than one IP address pool, MetalLB uses an available IP address from the higher priority IP address pool. If no IP addresses are available from the assigned IP address pools with a high priority, MetalLB uses available IP addresses from an IP address pool with lower priority or no priority. 
+
+[NOTE]
+====
+You can use the `matchLabels` label selector and/or the `matchExpressions` label selector for both the `namespaceSelectors` and `serviceSelectors` specifications. This example demonstrates one label selector for each specification.
```

Comment:
See IBM Style entry on `and/or`: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a